examples/alpha/README.md

```diff
@@ -0,0 +1,19 @@
+# alpha
+
+There are many options for manipulating the transparency of colors in your Tailwind config, but one straightforward method is to use alpha hex colors. These are part of the [new CSS spec](https://www.w3.org/TR/css-color-4/#hex-notation) and work by adding an alpha channel to the end of a hex value.
+
+For three-digit hex values, like `#fff`, this is a single value: `#fffc`. For six-digits, two values are added: `#ffffffcc`. Unfortunately, these alpha values must be written in hexadecimal, so something like 75% would actually be written as `bf`. To solve this, you can include an object translating these values.
+
+```js
+const a = require('@viget/tailwindcss-plugins/alpha'),
```

Comment:
I like the idea of requiring this at the top of the tailwind config. How about we move this from **examples** to **utilities**?

Once we get the path issue fixed for plugin requires, these would look like:

```
# for utilities, at the top
const a = require('@viget/tailwindcss-plugins/utilities/alpha'),

# for plugins, in plugins array
plugins: [
  require('@viget/tailwindcss-plugins/rect'),
]
```